src/index.js

```diff
@@ -2,6 +2,9 @@ import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import axios from 'axios'
+
+axios.defaults.baseURL = "http://localhost:3000/api";
```

Comment:
If you have deployed to your api, then that URL has to be here.

Reply:
Yep, we will do so. Thanks.